### PR TITLE
2FA should have different and much looser configuration than default …

### PR DESCRIPTION
**Pull request by matllubos**
…OTP key generator

### CODE REVIEW

**Review by zamazaljiri — APPROVED** [review]

**Review by Formulka — APPROVED** [review]

File: auth_token/utils.py [hunk 1]
```diff
@@ -416,7 +429,8 @@ def create_otp(slug, related_objects=None, data=None, key_generator=None, expira
     if deactivate_old:
         deactivate_otp(slug, related_objects=related_objects)
 
-    key_generator = import_string(settings.OTP_DEFAULT_KEY_GENERATOR) if key_generator is None else key_generator
+    key_generator = settings.OTP_DEFAULT_KEY_GENERATOR if key_generator is None else key_generator
```
Comment (rubickcz) [inline auth_token/utils.py:432]: Shouldn't here be used `AUTHORIZATION_OTP_BACKEND_DEFAULT_KEY_GENERATOR` ?

File: example/dj/apps/app/tests/utils.py [hunk 2]
```diff
@@ -724,3 +726,11 @@ def key_genertor():
         create_otp('test', key_generator=lambda: '1234')
         assert_equal(create_otp('test', key_generator=key_genertor).secret_key, '1235')
         assert_equal(key_genertor.iteration, 100)
+
+    def test_generate_otp_key_should_generate_20_chars_length_key(self):
+        for _ in range(20):
+            assert_true(re.match(r'^[a-zA-z0-9]{20}$', generate_otp_key()))
```
Comment (rubickcz) [inline example/dj/apps/app/tests/utils.py:732]: `A-z` -> `A-Z`

File: auth_token/utils.py (hunk 1 above)
Comment (matllubos) [reply auth_token/utils.py:432]: No this is create_otp function not authorization backend. There realy need to be default OTP generator. Maybe you didn't open hidden code

File: example/dj/apps/app/tests/utils.py (hunk 2 above)
Comment (matllubos) [reply example/dj/apps/app/tests/utils.py:732]: cool thanks

**Pull request merged**